configuration/configure-unhealthy-cluster-removal.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="configure-unhealthy-cluster-removal"]
+= Configuring automatic removal of unhealthy clusters
+include::modules/common-attributes.adoc[]
+:context: configure-unhealthy-cluster-removal
+
+toc::[]
+
+[role="_abstract"]
+{product-title} ({product-title-short}) provides the ability to automatically remove unhealthy clusters in your system. If this feature is enabled, when Central has been unable to reach Sensor in a cluster for a configured period of time, the cluster is considered unhealthy and is deleted (decommissioned). Automatically deleting unhealthy clusters allows you to more easily monitor only active clusters. You can configure cluster removal settings by using the {product-title-short} portal or the application programming interface (API).
+
+This setting is disabled by default. To enable this setting, enter a nonzero number in the *Decommissioned cluster age* field, as described in the following procedure. The *Decommissioned cluster age* field indicates the number of days that a cluster can remain unreachable before it is considered unhealthy. When a cluster is unhealthy, the Clusters page displays the status of the cluster and the number of days after which the cluster will be deleted if it continues to remain unhealthy. After a cluster is removed, the removal is documented with an `info` log in the Central logs.
```

Comment:
Consider adding a 1-liner info. about the 'Ignore clusters which have labels' setting in the introduction para., as it includes info. about 'customers can specify not to delete the unhealthy clusters even after they passed the decommissioning date'. This is significant info.